pkg/rt/subscriber.go

```diff
@@ -267,7 +293,71 @@ func (s *Subscriber) queryOrigins(ctx context.Context, ts uint64) (currentName s
 	case http.StatusUnauthorized, http.StatusForbidden:
 		result = apiResultError
 		level.Error(s.logger).Log("status_code", resp.StatusCode, "response_ts", response.Timestamp, "err", apiErr, "msg", "token may be invalid")
-		delay = 15 * time.Second
+		delay = 120 * time.Second
+
+	default:
+		result = apiResultUnknown
+		level.Error(s.logger).Log("status_code", resp.StatusCode, "response_ts", response.Timestamp, "err", apiErr)
+		delay = 5 * time.Second
```

Comment:
Maybe bump this a little higher to say 15 or 30?

Reply:
That's the existing default delay (just looks new because of the diff). I agree that this should be a little higher though and will bump it up to 30.